Dockerfile

```diff
@@ -9,36 +9,37 @@
 #   Red Hat, Inc. - initial implementation
 #
 
-FROM rust:1.49.0-alpine3.12 as newdoc-builder
-RUN cargo install newdoc
+FROM docker.io/library/rust:1.51-alpine3.13 as newdoc-builder
+RUN apk add --no-cache --update  musl-dev \
+    && cargo install newdoc
 
-FROM golang:1.15.6-alpine3.12 as htmltest-builder
+FROM docker.io/library/golang:1.15-alpine3.13 as htmltest-builder
```

Comment:
1.15 is a rolling tag so usually we stick to a tag that is not changing

Reply:
You mean to make sure we can rebuild exactly the same? 
I see this here as a feature to enable to build newer images including security fixes without changing the Dockerfile in master.

Reply:
yes, to have something that is reproduced all the time

about security fixes updates, probably that dependabot can provide such enhancements ?